public/app/AppWrapper.tsx

```diff
@@ -59,7 +59,7 @@ export class AppWrapper extends React.Component<AppWrapperProps, AppWrapperState
 
     return (
       <Route
-        exact
+        exact={route.exact === undefined ? true : route.exact}
```

Comment:
One of the great newish js features, fyi :smile_cat: 
```suggestion
        exact={route.exact ?? true}
```